catboost/libs/fstr/calc_fstr.h

```diff
@@ -130,20 +130,20 @@ TVector<TFeatureInteraction> CalcFeatureInteraction(
 
 TVector<TVector<double>> CalcInteraction(const TFullModel& model);
 TVector<TVector<double>> GetFeatureImportances(
-    const TString& type,
+    const EFstrType type,
```

Comment:
There's no need for const here and below.

Reply:
Is it OK to use non-const as a first argument? @annaveronika mentioned above that const parameters should go first